packages/local_auth/local_auth_darwin/lib/local_auth_darwin.dart

```diff
@@ -118,4 +122,26 @@ class LocalAuthDarwin extends LocalAuthPlatform {
       localizedFallbackTitle: messages?.localizedFallbackTitle,
     );
   }
+
+  AuthStrings _pigeonStringsFromMacOSAuthMessages(
+      String localizedReason, Iterable<AuthMessages> messagesList) {
+    MacOSAuthMessages? messages;
+    for (final AuthMessages entry in messagesList) {
+      if (entry is MacOSAuthMessages) {
+        messages = entry;
+        break;
+      }
+    }
+    return AuthStrings(
+      reason: localizedReason,
+      lockOut: messages?.lockOut ?? macOSLockOut,
+      goToSettingsButton: messages?.goToSettingsButton ?? goToSettings,
+      goToSettingsDescription:
+          messages?.goToSettingsDescription ?? macOSGoToSettingsDescription,
+      // TODO(stuartmorgan): The default's name is confusing here for legacy
```

Comment:
What legacy reasons require a new constant to have a confusing name?

Reply:
That was a comment that was copied from auth_messages_ios

Reply:
I'm aware of that. I wrote that comment when writing new code that used an existing constant that we couldn't change without breaking clients.

That context doesn't apply to a completely new constant, so I don't understand why the TODO and the confusing name were copied into this file.